### ISSUE TITLE
Nodejs Only

### ISSUE THREAD

**Issue by rsamhollyer** [wesbos/eslint-config-wesbos#106]
Sorry if this is the wrong place to ask, but is there any chance that this tool will only support Nodejs without any React tooling?
If so, great!
If not, what would I need to remove from this package that is specifically for React?

Thanks in advance and point me in the correct direction if I'm supposed to ask these questions elsewhere!

**Comment by wesbos** [comment]
Yeah not sure how to best approach this. Seems like id need to publish multiple packages, each with their own peer deps.. Something that needs to happen.. I think Brad Garropy did something similar so maybe we can check thatout?

**Comment by wesbos** [comment]
With the latest fix of not needing to install peerDeps, Ill be able to do this a lot more now
